modules/odc-creating-a-project-using-the-developer-perspective-in-the-web-console.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-creating-a-project-using-the-developer-perspective-in-the-web-console_{context}"]
+= Creating a project using the Developer perspective in the web console
+
+You can use the Developer perspective in the {product-title} web console to create a project in your namespace.
+
+.Prerequisites
+
+* Ensure that you have the appropriate roles and permissions to create projects, applications, and other workloads in {product-title}. See link:https://docs.openshift.com/container-platform/3.4/architecture/additional_concepts/authorization.html#roles[here] for more details on roles and permissions.
+
+.Procedure
+You can create a project using the *Developer* perspective, as follows:
+
+. In the *Add* view, click the *Project* drop-down to see a list of all available projects. Select *Create Project*.
++
+image::odc_create_project.png[Create Project]
+
+. In the *Create Project* dialog box, enter a unique name for the *Name* field. For this tutorial, enter `myproject` as the name of the project in the *Name* field.
```

Comment:
I avoid referring to our docs as individual tutorials, or anything similar. Perhaps rewrite this as:

    For example, enter `myproject` as the name of the project in the *Name* field.